src/components/NoteItems/AddNote.jsx

```diff
@@ -1,28 +1,56 @@
+import { useState } from "react";
 import { addNotes } from "../../utilities";
 import { useNote } from "../../contexts";
-
+import { ColorPalette } from "../../components";
+import "../../styles/main.css";
 import {
   LabelRoundedIcon,
   PushPinRoundedIcon,
+  PaletteRoundedIcon,
 } from "../../components";
 import "./add-note.css";
 
 const AddNote = () => {
   const emptyNote = {
     title: "",
     content: "",
+    noteColor: "",
+  };
+  const { note, setNote, noteState, noteDispatch } = useNote();
+  const [noteColor, setNoteColor] = useState("var(--cta-ultralight)");
+  const [noteLabel, setNoteLabel] = useState("no-label");
+
+  const getDate = () => {
+    const date = new Date();
+    const day = date.getDay();
+    const month = date.getMonth();
+    const year = date.getFullYear();
+    const hour = date.getHours() < 10 ? `0${date.getHours()}` : date.getHours();
+    const min =
+      date.getMinutes() < 10 ? `0${date.getMinutes()}` : date.getMinutes();
+    return `${day}/${month}/${year}  ${hour}:${min}`;
   };
-  const { note, setNote, noteDispatch } = useNote();
+
+  const noteDate = getDate();
 
   const addNoteHandler = (e) => {
     e.preventDefault();
-    if(!(note.title === "" && note.content === "")) {
-      addNotes(noteDispatch, {note: note});
+    if (!(note.title === "" && note.content === "")) {
+      addNotes(noteDispatch, {
+        note: {
+          ...note,
+          createdOn: noteDate,
+          noteColor: noteColor,
+          noteLabel: noteLabel,
+        },
+      });
       setTimeout(() => {
         setNote(emptyNote);
-      }, 1500)
+        setNoteColor("var(--cta-ultralight)");
+        setNoteLabel("no-label");
+      }, 1500);
     }
-  }
+  };
 
   return (
     <form
```

Comment:
![image](https://user-images.githubusercontent.com/75421090/162105532-1d41f81f-4633-49e1-9665-fd644db56bab.png)
Maybe there's some issue in loading these css classes in deploy preview

Is add-note-container not being declared anywhere ?

Reply:
Yes it is this is the reason, if you scroll down you'll see the colour picker right at the bottom of the screen! add-note-container class is vanishing on deployment. The archive thing is because of mockbee, the api permanently deletes the whole note if you delete from archive, doesn't add to trash :(